crates/storage/src/collections/binary_heap/mod.rs

```diff
@@ -106,7 +106,9 @@ where
     ///
     /// # Example
     ///
-    /// ```
+    /// ```ignore
+    /// # // TODO [#XYZ]: We currently ignore this test since we stopped exposing the `collections`
+    /// # // module publicly.
```

Comment:
But from within the `collections` module (here) you should still have access?

Reply:
No, the doc tests get treated as if they were in a separate crate (kinda like the `tests` directory), so we don't have access to private members